packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.js

```diff
@@ -1,14 +1,22 @@
 const fs = require('fs');
+const path = require('path');
 const config = require('@venncity/nested-config')(__dirname);
 const { importSchema } = require('graphql-import');
 const { DefaultParser, DatabaseType } = require('prisma-datamodel');
 const { makeExecutableSchema } = require('graphql-tools');
 const { graphqlSync, introspectionQuery } = require('graphql');
 
 const parser = DefaultParser.create(DatabaseType.postgres);
-const openCrudDataModel = parser.parseFromSchemaString(fs.readFileSync(config.get('graphql.schema.datamodel.path'), 'utf8'));
+const dataModelPath = config.has('graphql.schema.datamodel.path')
+  ? config.get('graphql.schema.datamodel.path')
+  : path.join(__dirname, '../../test/datamodel.graphql');
 
-const openCrudSchemaGraphql = importSchema(config.get('graphql.schema.sdl.path'));
+const openCrudDataModel = parser.parseFromSchemaString(fs.readFileSync(dataModelPath, 'utf8'));
+
+const schemaSdlPath = config.has('graphql.schema.sdl.path')
+  ? config.get('graphql.schema.sdl.path')
+  : path.join(__dirname, '../../test/openCRUD.graphql');
+const openCrudSchemaGraphql = importSchema(schemaSdlPath);
```

Comment:
Just a reminder fo my self, would like to understand better the diff between the SDL and the data model files